### PR TITLE
meson: Fixes

### PR DESCRIPTION
**Pull request by medhefgo**

### CODE REVIEW

File: src/kernel-install/meson.build [hunk 1]
```diff
@@ -12,10 +12,13 @@ loaderentry_install = custom_target(
...
 
-uki_copy_install = files('90-uki-copy.install')
+kernel_install_files = files(
+        '50-depmod.install',
+        '90-uki-copy.install',
+)
 
 if want_kernel_install
-        install_data('50-depmod.install',
+        install_data(kernel_install_files,
```
Comment (keszybz) [inline src/kernel-install/meson.build:21]: Yikes!

File: catalog/meson.build [hunk 2]
```diff
@@ -1,18 +1,24 @@
...
         'systemd.it.catalog',
+        'systemd.ko.catalog',
         'systemd.pl.catalog',
         'systemd.pt_BR.catalog',
         'systemd.ru.catalog',
+        'systemd.sr.catalog',
         'systemd.zh_CN.catalog',
         'systemd.zh_TW.catalog',
-        'systemd.catalog']
+]
```
Comment (keszybz) [inline catalog/meson.build:21]: WAT1

File: network/meson.build [hunk 3]
```diff
@@ -3,6 +3,7 @@
 if conf.get('ENABLE_NETWORKD') == 1
         install_data('80-6rd-tunnel.network',
                      '80-container-host0.network',
+                     '80-container-vb.network',
```
Comment (keszybz) [inline network/meson.build:6]: Uuuuuh.

File: rules.d/meson.build [hunk 4]
```diff
@@ -11,6 +11,7 @@ rules = [
                '60-drm.rules',
                '60-evdev.rules',
                '60-fido-id.rules',
+               '60-infiniband.rules',
```
Comment (keszybz) [inline rules.d/meson.build:14]: Jesus.

**Review by keszybz — APPROVED** [review]
LGTM.

**Comment by keszybz** [comment]
mkosi / ci (centos, 9):
```
Run sudo mkosi shell bash -c "[[ -e /testok ]] || { cat /failed-services; exit 1; }"
● auditd.service loaded failed failed Security Auditing Service
```
I'm pretty sure this is related to 2aba77057e55a1082296c10f61e19d8c5e1eb1f7.

CentOS CI (Arch Linux):
```
19:18:39 #3  0x00007fd363da8697 in log_assert_failed (
19:18:39     text=text@entry=0x559a6c5cfaa8 "y > usec_sub_unsigned(x, 2 * USEC_PER_DAY) && y < usec_add(x, 2* USEC_PER_DAY)", 
19:18:39     file=file@entry=0x559a6c5cefd1 "src/test/test-time-util.c", 
19:18:39     line=line@entry=371, 
19:18:39     func=func@entry=0x559a6c5d0220 <__func__.12> "test_format_timestamp")
```
#26172 

rpm-build:fedora-rawhide-i386:
` - Status code: 404 for https://kojipkgs.fedoraproject.org/repos/rawhide/latest/i386/repodata/e41763b2a5b7d10df5ec7dbcc83ec4dac514821478b1ee3c66502f6bde49eefd-filelists.xml.gz (IP: 38.145.60.21)`
At least for once it's not our fault.

**Comment by DaanDeMeyer** [comment]
> I'm pretty sure this is related to https://github.com/systemd/systemd/commit/2aba77057e55a1082296c10f61e19d8c5e1eb1f7.

We recently changed the default kernel command line arguments of mkosi to not disable audit by default anymore which is what triggered this I think. But already fixed in a newer commit

**Pull request merged**